pkg/redirect/redirect_test.go

```diff
@@ -90,6 +90,26 @@ func TestPrefixlessHosts(t *testing.T) {
 				all, _ := io.ReadAll(resp.Body)
 				t.Errorf("got error %v, want %v; %s", gotErr, c.wantErr, string(all))
 			}
+
+			link := resp.Header.Get("Link")
+			if strings.Contains(link, `; rel="next"`) {
+				t.Logf("got Link next header: %s", resp.Header.Get("Link"))
+				next := strings.TrimPrefix(link, "<")
+				next = next[:strings.Index(next, ">")]
```

Comment:
if somebody sends a whack ass Link header could this cause a panic?

Reply:
Maybe, but it'd probably cause a lot more havoc for everyone so they'd roll it back. In any case it'd only make our test fail 🤷 

But I added a check that the value includes a `>` at least, so we're not panic-prone. A drastic non-conformant change to the link header format would still cause us (and everyone) problems.